### ISSUE TITLE
"template group" node configuration

### ISSUE THREAD

**Issue by PhysicApp** [BALKANGraph/OrgChartJS#782]
hello, good morning, I need the node to adjust based on the elements inside the "text" tag so that it is centered inside the node.

[https://code.balkan.app/template-group#JS](url)

**Comment by ZornitsaPesheva** [comment]
Check if this demo can help:
https://code.balkan.app/org-chart-js/list-in-groups#JS

**Comment by PhysicApp** [comment]
Yes, but I need the node to be custom, but when the custom loses what it adjusts automatically, and I need when the group is expanded that the nodes have the template defined at the beginning in the org chart object.

example:
https://code.balkan.app/template-group#JS

**Comment by ZornitsaPesheva** [comment]
You should customize the group without fixing the with and height of the node;
https://code.balkan.app/org-chart-js/custom-group-with-list#JS
Here is the group template:
https://balkan.app/OrgChartJS/Docs/PredefinedTemplates#groupTemplate
Click of **[Get the source code of group template](https://balkan.app/OrgChartJS/Docs/PredefinedTemplates#)**

**Comment by PhysicApp** [comment]
I want that when the group node is maximized, nodes inside it take the template defined in the organization chart object.

I've been trying and that's what I got, any idea how to do it?

https://code.balkan.app/template-group-nad#JS

**Comment by PhysicApp** [comment]
I need that when clicking on the group it expands, and shows the nodes to be displayed normally:

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/32747854-f1d9-4526-b408-2eaa5df56491)

example:

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/f2b5567d-15c3-47bb-b11f-ff5ec16dca6c)

**Comment by ZornitsaPesheva** [comment]
So on minimized you want the first screenshot and on maximized - the second screenshot?

**Comment by PhysicApp** [comment]
Yes, when you click on the group, change the template as shown in the second screenshot.

**Comment by ZornitsaPesheva** [comment]
I have created this code demo for you:
https://code.balkan.app/org-chart-js/min-max-list-to-nodes#JS

**Comment by PhysicApp** [comment]
everything is correct, only the only detail is that the nodes that are not grouped are deconfigured.

https://code.balkan.app/template-group-prefinal#JS

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/5160f0c1-fc72-4244-af9e-9d95e71b39ca)

**Comment by ZornitsaPesheva** [comment]
You should have a condition to minimize only the nodes with "group" template.
Changed the code demo:
https://code.balkan.app/org-chart-js/min-max-list-to-nodes#JS

**Comment by PhysicApp** [comment]
When expanding overlapping groups, with layout scaleInitial:OrgChart.match.width , how can I avoid it?

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/81544ea6-d338-4c9f-8e50-f978c954efba)

https://code.balkan.app/template-group-prefinal#JS

**Comment by ZornitsaPesheva** [comment]
I have edited your example trying to reproduce the issue, but still cannot get the result of your screenshot:
https://code.balkan.app/org-chart-js/issue-782#JS
![image](https://github.com/BALKANGraph/OrgChartJS/assets/19305472/c0115144-46ba-4824-8e5c-2df5d827708f)

**Comment by PhysicApp** [comment]
I have a problem, I have a relationship with "slink" of a node that is in a grouping and when I want to expand the grouping it doesn't expand and it doesn't let me see the relationship:

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/96b3c40d-2977-47bf-95d0-1420d733ef53)

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/d4ec558a-c594-4718-b465-726bc62ae041)

Error en consola:
![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/97d61992-3e65-47a4-b24d-24dbaf3ccc2f)

**Comment by ZornitsaPesheva** [comment]
We don't support slinks between different trees/subtrees(groups)
You can use clinks instead.

**Comment by PhysicApp** [comment]
Yes, but I don't want it to go through the nodes, is there a way to avoid it?

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/3fdfe70c-5546-4dde-ab20-20ab6d458388)

**Comment by ZornitsaPesheva** [comment]
You can try to change the curve:
https://code.balkan.app/org-chart-js/clink_curve#JS

**Comment by PhysicApp** [comment]
> Edité su ejemplo tratando de reproducir el problema, pero aún no puedo obtener el resultado de su captura de pantalla: https://code.balkan.app/org-chart-js/issue-782#JS ![imagen](https://user-images.githubusercontent.com/19305472/259929389-c0115144-46ba-4824-8e5c-2df5d827708f.png)

The error appears when selecting the design indicated in the image:

![image](https://github.com/BALKANGraph/OrgChartJS/assets/84802936/3bb5e222-0bec-4d3b-9679-8f6099e2e7c7)

**Comment by ZornitsaPesheva** [comment]
Yes, I was able to isolate the issue in a more simple example: 
https://code.balkan.app/org-chart-js/tree-layout-group-issue.#JS
Thank you for reporting it.

**Comment by PhysicApp** [comment]
> I have a problem, I have a relationship with "slink" of a node that is in a grouping and when I want to expand the grouping it doesn't expand and it doesn't let me see the relationship:
> 
> ![image](https://user-images.githubusercontent.com/84802936/260168048-96b3c40d-2977-47bf-95d0-1420d733ef53.png)
> 
> ![image](https://user-images.githubusercontent.com/84802936/260168282-d4ec558a-c594-4718-b465-726bc62ae041.png)
> 
> Console error: ![image](https://user-images.githubusercontent.com/84802936/260168403-97d61992-3e65-47a4-b24d-24dbaf3ccc2f.png)
> 
> Example: https://code.balkan.app/template-group-prefinal#JS

> You can try to change the curve: https://code.balkan.app/org-chart-js/clink_curve#JS

It will be that it implements this functionality at some point, is that this functionality is required

**Comment by PhysicApp** [comment]
> We don't support slinks between different trees/subtrees(groups) You can use clinks instead.

**Comment by PhysicApp** [comment]
> > I have a problem, I have a relationship with "slink" of a node that is in a grouping and when I want to expand the grouping it doesn't expand and it doesn't let me see the relationship:
> > ![image](https://user-images.githubusercontent.com/84802936/260168048-96b3c40d-2977-47bf-95d0-1420d733ef53.png)
> > ![image](https://user-images.githubusercontent.com/84802936/260168282-d4ec558a-c594-4718-b465-726bc62ae041.png)
> > Console error: ![image](https://user-images.githubusercontent.com/84802936/260168403-97d61992-3e65-47a4-b24d-24dbaf3ccc2f.png)
> > Example: https://code.balkan.app/template-group-prefinal#JS
> 
> > We don't support slinks between different trees/subtrees(groups) You can use clinks instead.
> 
> It will be that it implements this functionality at some point, is that this functionality is required
